documentation/configuration.md

```diff
@@ -135,6 +135,12 @@ default: /tmp
 Set spider logs directory. All spiders have their own dedicated log file
 stored under the `log_dir` folder.
```

Comment:
Add in a reference to the `log_to_file` config. "Ignored if logging not enabled"

Reply:
Done